src/metabase/lib/stage.cljc

```diff
@@ -143,10 +143,13 @@
                                                            (lib.util/query-stage query previous-stage-number))
            :let [source-alias (or ((some-fn :lib/desired-column-alias :lib/source-column-alias) col)
                                   (lib.metadata.calculation/column-name query stage-number col))]]
-       (-> col
-           (assoc :lib/source               :source/previous-stage
-                  :lib/source-column-alias  source-alias
-                  :lib/desired-column-alias (unique-name-fn source-alias))
+       (-> (merge
+            col
+            {:lib/source               :source/previous-stage
+             :lib/source-column-alias  source-alias
+             :lib/desired-column-alias (unique-name-fn source-alias)}
+            (when (:metabase.lib.card/force-broken-id-refs col)
+              (select-keys col [:metabase.lib.card/force-broken-id-refs])))
```

Comment:
@camsaul What do these lines achieve? Isn't this property included in line 147 anyway?